assets/src/edit-story/app/api/apiProvider.js

```diff
@@ -35,7 +35,7 @@ import Context from './context';
 
 function APIProvider({ children }) {
   const {
-    api: { stories, media, link, users },
+    api: { stories, media, link, users, status },
```

Comment:
There's also the post status using `status`, should we perhaps rename this for clarity? E.g. `restStatus` WDYT?